exchanges/bybit/bybit_cfutures.go

```diff
@@ -1089,20 +1090,22 @@ func (by *Bybit) GetTradingFeeRate(ctx context.Context, symbol currency.Pair) (t
 	params := url.Values{}
 	resp := struct {
 		Result struct {
-			TakerFeeRate float64 `json:"taker_fee_rate,string"`
-			MakerFeeRate float64 `json:"maker_fee_rate,string"`
+			TakerFeeRate convert.StringToFloat64 `json:"taker_fee_rate"`
```

Comment:
More of a question for you. There is a `user_id` field returned in this reponse. Is it worth adding? If so, it would be easier to just return the response struct than the seperation of taker maker. At least then the user will understand which one is taker and which one is maker without needing to do any checks or inspection of this function. Idk whether they'll appreciate the userid metadata, but it is part of the response struct is all

Reply:
Dunno, added it anyway. It's a V2 endpoint the others from v3 onwards doesn't have it. 🤷